bad_connector_woocommerce/models/stock_picking/common.py

```diff
@@ -0,0 +1,82 @@
+import logging
+
+from odoo import api, fields, models
+
+from odoo.addons.component.core import Component
+
+_logger = logging.getLogger(__name__)
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    woo_bind_ids = fields.One2many(
+        comodel_name="woo.stock.picking.refund",
+        inverse_name="odoo_id",
+        string="WooCommerce Bindings(Stock)",
+        copy=False,
+    )
+    is_refund = fields.Boolean(string="Refund Quantity With Amount")
+    is_return_picking = fields.Boolean(
```

Comment:
In the delivery module, it has the same field name is_return_picking, it might get a problem if we include the delivery module in the database, your suggestion please RUS, should we rename this field?